dist/index.js

```diff
@@ -124,19 +124,17 @@ function koaHotware(compiler, options) {
           switch (_context2.prev = _context2.next) {
             case 0:
               stream = new _stream.PassThrough();
-
```

Comment:
in the future please don't commit the `dist` directory. won't let that block this PR this time around.